### DISCUSSION TITLE
branchPrefix gets trimmed?

### DISCUSSION THREAD

**Discussion by pahu-dev** [renovatebot/renovate#19186] [Request Help] [answered]
Hi,
Not sure if it is a bug or not, but couldn't find anything in the documentation that refers to it.
I'm trying to change the: `branchPrefix: 'renovate--'` instead of the default `renovate/` but it results in `renovate-` (the 2nd `-` got trimmed).
I've also tried to set:
```
{ branchPrefix: 'renovate-', additionalBranchPrefix: '-' }
```
Which gave the same unintended result of `renovate-`.
Is this a bug, or is there a reason behind it?

When I set the onboarding branch to `renovate--configure` it works as expected.

Thank you.

**Comment by viceice** [comment, marked as answer]
expected, we explicit trim multiple dashes.
